dcos-log/journal/reader/read.go

```diff
@@ -219,3 +220,68 @@ func (r *Reader) Close() error {
 	}
 	return r.Journal.Close()
 }
+
+// Follow is a wrapper function, which can be called multiple times to mimic a journal tailing.
+func (r *Reader) Follow(wait time.Duration, writer io.Writer) error {
+	n, err := io.Copy(writer, r)
+	if err != nil && err != io.EOF {
+		return err
+	}
+
+	// if the number of read lines more then 0, we did not reach the journald bottom and can exit early
```

Comment:
Is this the number of read lines or the number of read bytes?

Reply:
this one is a number of read lines